src/components/FeaturedContent/FeaturedContent.test.tsx

```diff
@@ -0,0 +1,221 @@
+import { render, screen } from "@testing-library/react";
+import { axe } from "jest-axe";
+import renderer from "react-test-renderer";
+
+import FeaturedContent from "./FeaturedContent";
+import Button from "../Button/Button";
+import Heading from "../Heading/Heading";
+import Text from "../Text/Text";
+export const textContent = (
+  <>
+    <div>
+      <br />
+      <Text size="overline1"> Featured </Text>
+      <Heading level="h1"> Spotlight on the Public Domain </Heading>
```

Comment:
We _just_ added this recently and didn't bring it up but you can get a similar style if you use the `overline` prop in the `Heading` component.

```
<Heading level="h1" overline="Featured">Spotlight....
```